pkgs/development/python-modules/django_appconf/default.nix

```diff
@@ -8,10 +8,11 @@ buildPythonPackage rec {
     sha256 = "35f13ca4d567f132b960e2cd4c832c2d03cb6543452d34e29b7ba10371ba80e3";
   };
 
-  # No tests in archive
+  # No tests in pypi archive, 1.0.3 doesn't appear to be present in github repo
+  # https://github.com/django-compressor/django-appconf/issues/48
```

Comment:
https://github.com/django-compressor/django-appconf/releases/tag/1.0.3 ?

Reply:
Ahh all the other releases are tagged with `v` prefixes, this one's just different. Thanks for checking - will fix.